### DISCUSSION TITLE
[Feature] Map stack open in a gallery of selected pfotos instead of a slideshow

### DISCUSSION THREAD

**Discussion by jojo-gfx** [immich-app/immich#12108] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

At time open stacks from the map as a sort of slideshow, but that is not, what users want in a photo-archive, because it's a torture to find photos in this way (have stacks with over 100 photos on map).

It would be nice immich could open the staple in a normal gallery overview like an album where only be selected the photos from staple.

That would be incredible! :)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4519

Please look for duplicates before opening an FR, as requested in the template.

**Reply by jojo-gfx** [reply]
Ok, not so easy mostly to find. 😅
